.github/workflows/unit-tests.yml

```diff
@@ -6,7 +6,9 @@ jobs:
     strategy:
       matrix:
         startarg:
-          - { make: 'check-memory', cc: 'clang', tag: 'i386' }
+          # new Clang breaks Valgrind, so use gcc for all `check-memory`
+          # scenarios for now: https://bugs.kde.org/show_bug.cgi?id=452758
```

Comment:
Alternatively:
```
            # HACK: valgrind does not work with DWARF5 symbols generated by Clang 14 https://bugs.kde.org/show_bug.cgi?id=452758
            command = ['sh', '-c', 'CFLAGS="-O2 -gdwarf-4" /source/autogen.sh; exec make -j$(nproc) ' + shlex.quote(args.make)]
```

Reply:
...thus also avoiding the test name changing.  Sounds good :)